src/pulse3D/plate_recording.py

```diff
@@ -373,6 +421,35 @@ def __next__(self):
             raise StopIteration
 
 
+def load_files(path):
+    with tempfile.TemporaryDirectory() as recording_dir:
+        # get file names
+        if path.endswith('.zip'):
+            zf = zipfile.ZipFile(path)
+            zf.extractall(path=recording_dir)
+        else:
+            recording_dir = path
+        # remove folder that gets created when zipped on MacOS
+        folders = os.listdir(recording_dir)
+        if "__MACOSX" in folders:
+            folders.remove("__MACOSX")
+        # sort files
+        h5_files = glob.glob(os.path.join(recording_dir, folders[0], "*.h5"))
```

Comment:
I don't think this does what you want, the path being passed to glob will be something like `/some_tmp_dir/MA201110001__2020_09_03_213024__A6.h5/*.h5`  if there are only .h5 files in the unzipped dir

if it was
```
h5_files = glob.glob(os.path.join(recording_dir, '*.h5'))
```
then it would only return a list of .h5 files, then I don't think you'd need to check for a `__MACOSX` folder either since glob would skip it

Reply:
The test must be passing then because I zipped the entire folder instead of just the files in the folder. I just tried the latter and it fails as you described. Think I found a simpler way to address handle both cases

Reply:
glob can recursive the directories by using '**' in the path
```
glob.glob(os.path.join(recording_dir, '**', '*.ht'), recursive=True)
```
will grab all .h5 files recording_dir and below